webapp-django/package.json

```diff
@@ -10,31 +10,31 @@
   "license": "MPL-2.0",
   "devDependencies": {
     "cssmin": "0.4.3",
-    "eslint": "4.19.1",
-    "eslint-config-prettier": "2.9.0",
-    "eslint-plugin-prettier": "2.6.0",
-    "graceful-fs": "4.1.11",
-    "less": "3.0.2",
-    "prettier": "1.13.2",
-    "uglify-js": "3.3.23"
+    "eslint": "7.13.0",
+    "eslint-config-prettier": "6.15.0",
+    "eslint-plugin-prettier": "3.1.4",
+    "graceful-fs": "4.2.4",
+    "less": "3.12.2",
+    "prettier": "2.1.2",
+    "uglify-js": "3.11.5"
   },
   "dependencies": {
     "@sentry/cli": "^1.40.0",
     "Select2": "github:select2/select2#3.5.4",
```

Comment:
I couldn't update Select2. I think we have to make some code changes to make it work. I wrote up a bug for that.